sql/scan.go

```diff
@@ -386,11 +386,19 @@ func (n *scanNode) initOrdering() {
 }
 
 // computeOrdering computes the ordering information for the specified set of
-// columns.
+// columns. The returned slice will be the same length as the input slice. If a
+// column is not part of the output 0 will be returned in the corresponding
+// field of the output. Consider the query:
+//
+//   SELECT v FROM t WHERE k = 1
+//
+// If there is an index on (k, v) and we're asking for the ordering for those
+// columns, computeOrdering will return (0, 1). This indicates that column k is
+// not part of the output column set and column v is in ascending order.
```

Comment:
A negative value indicates descending ordering. This doesn't appear in this code because indexes are currently always in ascending ordering. You can see the negative ordering values in `planner.orderBy` and `indexInfo.analyzeOrdering`.